src/reducers/appReducers.js

```diff
@@ -25,19 +26,19 @@ let initialState: State = {
 
 function appReducers(state: State = initialState, action: Action) {
   switch(action.type) {
-    case 'LOADING':
+    case LOADING:
```

Comment:
with proper flow typing, you can probly get away without using the constants. I'm not sure though if that is possible with normal flow outside of the fb setup

Reply:
I wouldn't recommend trying to get that to work though lol